features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/modules/statistics/usage.jag

```diff
@@ -9,17 +9,33 @@ var gson=com.google.gson.Gson();
 function getProviderAPIVersionUsage(providerName, apiName, server) {
     var usage = [];
 
+    if(usageClient == null){
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/d7e56b1d85254369d07aab9572f13d4c2120df97